server/api/credit_card.py

```diff
@@ -11,4 +11,4 @@
 # Route for getting all details of credit card transactions
 @creditcard_api.route('/creditcards/<int:card_holder_id>/transactions', methods=['GET'])
 def get_credit_card_transactions(card_holder_id):
-    return credit_card_controller.get_credit_cards_by_card_holder(card_holder_id)
+    return credit_card_controller.get_credit_card_transactions(card_holder_id)
```

Comment:
This is a confusing change - before i would expect this method to return cred_cards, now we're returning credit card transactions? Some type hints would help in general. 

What was this method return before - i do see the route is /transactions, was that always returning credit cards?